.github/workflows/python-publish.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       run: |
         python -m build --sdist --wheel --outdir dist/ .
     - name: Publish package
-      uses: pypa/gh-action-pypi-publish@v1.9.0
+      uses: pypa/gh-action-pypi-publish@v1.10.3
```

Comment:
I have the same question as in PR https://github.com/eduNEXT/eox-hooks/pull/60#discussion_r1820999140

Reply:
https://github.com/eduNEXT/eox-hooks/pull/60#discussion_r1821067634